GDJS/Runtime/scenestack.ts

```diff
@@ -67,8 +67,116 @@ namespace gdjs {
           this.replace(currentScene.getRequestedScene());
         } else if (request === gdjs.SceneChangeRequest.CLEAR_SCENES) {
           this.replace(currentScene.getRequestedScene(), true);
-        } else {
-          logger.error('Unrecognized change in scene stack: ' + request);
+        }
+
+        if (currentScene.getIsLoadingRequest()) {
+          currentScene.requestLoad(false);
+
+          if (currentScene._loadVariable) {
+            try {
+              const allSyncData =
+                currentScene._loadVariable.toJSObject() as GameSaveState;
```

Comment:
let's not access a private variable from another object here (doing `currentScene._loadVariable`)

you're doing the same for `_loadStorageName` below.

I think that instead of having 2 props in runtimescene, you should have 1 object:
`_loadRequestOptions = {
   storageName: ?string,
   storageVariable: ?Variable
}` 

And you can get it with a method `getLoadingRequestOptions()` - something like that